src/blog/BlogSection.jsx

```
@@ -0,0 +1,59 @@
+        <div className="cardTitle">{current?.title || "No posts yet"}</div>
+
+        <div className="tagRow" style={{ marginBottom: 10 }}>
+          {current?.tags?.map((t) => (
+            <span className="tag" key={t}>
+              {t}
+            </span>
+          ))}
+        </div>
+
+        {current ? (
+          <div className="blogHtml" dangerouslySetInnerHTML={{ __html: current.html }} />
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)